src/basic/calendarspec.c

```diff
@@ -876,6 +884,53 @@ static int parse_calendar_time(const char **p, CalendarSpec *c) {
         return r;
 }
 
+static int is_valid_timezone(char *tz, bool *result)
+{
+        pid_t pid;
+
+        if (tz == NULL || strlen(tz) == 0) {
+                *result = false;
+                return 0;
+        }
+
+        bool *shared = mmap(NULL, sizeof *shared, PROT_READ|PROT_WRITE, MAP_SHARED|MAP_ANONYMOUS, -1, 0);
+
+        if (shared == MAP_FAILED) {
+                return log_error_errno(errno, "mmap() failed: %m");
+        }
+
+        pid = fork();
+
+        if (pid == -1) {
+                munmap(shared, sizeof *shared);
```

Comment:
while we don#t do that for much of the older code, for new code please explicitly cast the return value of syscalls whose return value you want to ignore to (void), so that static analyzers understand this